### PR TITLE
Version 2 Vivaann Welfare Association

### PR DESCRIPTION
**Pull request by abhirajsinghchauhan**
Updated the styling and team member details for the batch 2020.

### CODE REVIEW

**Review by ajitzero — CHANGES_REQUESTED** [review]
Restore these files:
- `.gitignore`
- `LICENSE`

File: README.md [hunk 1]
```diff
@@ -13,7 +13,11 @@
 
 This repository contains the source code for the official Vivaann Welfare Association website.
 
-> Designed & built by [Ajit Panigrahi](https://twitter.com/ajitzero). Hosted on [GitHub Pages](https://pages.github.com/).
+Version 1 Designed & built by [Ajit Panigrahi](https://twitter.com/ajitzero).
```
Comment (ajitzero) [inline README.md:16]: Remove my name and add:

"Currently maintained and managed by [Abhiraj Singh Chauhan](https://www.linkedin.com/in/abhirajsinghchauhan/)"

Add your name in the release (I'll add that once this gets merged)

File: index.html [hunk 2]
```diff
@@ -1,330 +1,355 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
```
Comment (ajitzero) [inline index.html:2]: revert `lang="en"`

File: index.html [hunk 3]
```diff
@@ -1,330 +1,355 @@
...
 <head>
 	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
Comment (ajitzero) [inline index.html:5]: These `<meta>` tags are necessary for mobile view. I don't think your new website works for mobile view. Please work on that, as a new website should not break anything that used to work previously.

File: style.css [hunk 4]
```diff
@@ -1,258 +1,104 @@
-@charset "UTF-8";
```
Comment (ajitzero) [inline style.css:1]: Add the `@charset` back

**Review by ajitzero — CHANGES_REQUESTED** [review]
Also, don't use HEIC files like `img/bhargav-error.HEIC`

**Comment by ajitzero** [comment]
@abhirajsinghchauhan Thank you for the effort put in this new website. Please ensure these things are met:
- Website should work well on mobile (see [comment](https://github.com/vivaann/vivaann.github.io/pull/3#discussion_r668726304))
- Images should be optimized: use standard web file formats like PNG or JPEG and ensure they are less than 1MB.
- Maintain decent lighthouse scores and fix basic issues (like [this](https://github.com/vivaann/vivaann.github.io/pull/3#discussion_r668724674))

A suggestion: add a separate "Our Team" page for the full list of members and only show few names on the home page. Please add a section to show the members by year, so that at least we can mention the seniors who built Vivaann before us.

**Comment by abhirajsinghchauhan** [comment]
Hey @ajitzero. 
I appreciate your time and thanks for all the reviews you made. I am going to make a new pull request soon. As of now, I am closing this PR.

**Pull request closed without merging**